.goreleaser.yml

```diff
@@ -21,9 +21,37 @@ archives:
         format: zip
 dockers:
   - image_templates:
-      - itzg/mc-monitor:latest
-      - itzg/mc-monitor:{{ .Tag }}
+      -  itzg/mc-monitor:latest-amd64
```

Comment:
I would prefer these to follow the `{{ .Version }}-amd64` convention I'm using in my other projects:

https://github.com/itzg/mc-router/blob/a378eb6e094dfe4a43c4978dc3dbc5d9ca15fb81/.goreleaser.yml#L43-L84